### ISSUE TITLE
Incorrect load order resulting in crash

### ISSUE THREAD

**Issue by Bandifighter** [loot/skyrimse#749]
Hello,

If I let LOOT organise my mods, my game pretty much instantly crashes (sometimes it loads but after walking around e,g, in Riften for a minute it still crashes - sometimes other places too, I imagine it depends on if my game loads at the moment, - however it usually crashes even before that, on save load).

If I add custom conditions (make Bijin_AIO 2018.esp load after The Ordinary Women.esp and Qw_TheOrdinaryWomen_AIOverhaul_CRF Patch.esp) this issue is resolved and everything works fine without crashing.

So, in short - from my userlist.yaml:
  - name: 'Bijin_AIO 2018.esp'
    after:
      - 'Qw_TheOrdinaryWomen_AIOverhaul_CRF Patch.esp'
      - 'The Ordinary Women.esp'

Mods in question:
https://www.nexusmods.com/skyrimspecialedition/mods/11
https://www.nexusmods.com/skyrimspecialedition/mods/12376
https://www.nexusmods.com/skyrimspecialedition/mods/18369

**Comment by MacSplody** [comment]
What order did you install the mods in?

[The Ordinary Women SSE](https://www.nexusmods.com/skyrimspecialedition/mods/12376)
[Bijin NPCs SE](https://www.nexusmods.com/skyrimspecialedition/mods/11287)
[Bijin Warmaidens SE](https://www.nexusmods.com/skyrimspecialedition/mods/1825)
[Bijin Wives SE](https://www.nexusmods.com/skyrimspecialedition/mods/11247)

The install order must match the load order as both mods have loose files.

Adding that after won't actually solve the conflicts and you may still have issues elsewhere. It would also prevent anyone who prefers `The Ordinary Women.esp` changes, from using them without manually changing the order.

_Bijin AIO SE for USSEP_ should be used with _QUASIPC_, as it has patches for it.
You can find it on any of the [official Bijin](https://www.nexusmods.com/skyrimspecialedition/users/2650523?tab=user+files) mod pages. 
If you still want to use Bijin All in One 2019 then you'll need to make your own patches for it.

**Comment by Bandifighter** [comment]
That's the order for those mods in my MO2 already.
I prefer to allow Bijin AIO to overwrite Ordinary Women.
Bijin AIO already has a somewhat new version (from May) with AI Overhaul integrated so the patch from QUASIPC is not needed (and not even compatible).

**Comment by MacSplody** [comment]
> That's the order for those mods in my MO2 already.

That's why you had the crash, the assets didn't match up to the plugin order.
LOOT isn't going to account for the order in which you install loose files.

> I prefer to allow Bijin AIO to overwrite Ordinary Women.

Which is why you can change the order with a user rule. 
If an after is added to the masterlist, then no one can change the order through LOOT.

**Comment by Bandifighter** [comment]
![image](https://user-images.githubusercontent.com/28756193/63547875-f3208500-c524-11e9-9247-10dbfb8d9c5a.png)

> That's why you had the crash, the assets didn't match up to the plugin order.

Well, it doesn't match if I let LOOT do it (see pic) - but if I make it so that the Bijin AIO plugin loads after the Ordinary Womens' and its patch it works.
I'm confused as to why LOOT keeps putting the Bijin AIO plugin before Ordinary Women by default then, if one needs to manually adjust it (and overwrite with a user rule) so that it matches their loose file load order anyway? 
Are there some limitations or a reason I'm ignorant of?

**Comment by MacSplody** [comment]
> I'm confused as to why LOOT keeps putting the Bijin AIO plugin before Ordinary Women by default then, if one needs to manually adjust it (and overwrite with a user rule) so that it matches their loose file load order anyway?
Are there some limitations or a reason I'm ignorant of?

Because LOOT isn't looking at the loose files, it's comparing the two plugins and sorting them based on their record content. Below is an excert from the LOOT [Read the Docs](https://loot.readthedocs.io/en/latest/app/introduction.html) introduction.

> While LOOT is able to calculate correct load order positions for the vast majority of plugins using only their content, it cannot do so for all plugins. As such, LOOT provides a mechanism for supplying additional plugin metadata so that it may sort them correctly. An online masterlist is provided to supply metadata for many plugins that need it, while users can make their own metadata additions, which are saved in their userlist.

In this case it's not a sorting issue, as either order should work, but it requires the user to install the files in the correct order. I think a message could be added for NPC appearance mods.
